### ISSUE TITLE
Not able to go into live edit mode

### ISSUE THREAD

**Issue by lightningRalf** [ted-marozzi/mermaid-flow-feedback#11]
**Describe the bug**
erDiagram that shows a preview, but no way to live edit.

**To Reproduce**
So I tried it:
---
title: "ART"
---
erDiagram
  LLM ||--o{ ART : "is enhanced by"
  FEWSHOT-ZEROSHOT ||--o{ ART : "is improved by"
  COT ||--o{ ART : "is generated by"
  TOOLS ||--o{ ART : "is selected and integrated by"
  TASKLIBRARY ||--|{ ART : "is used by"
  PERFORMANCE ||--|| ART : "is improved by"
  PERFORMANCE }o--|| TASKS : "is measured on"

It shows the preview, but I cannot live edit my version.
Will this be developed? or am I doing something wrong?

**Expected behavior**
I can push a button, or see it directly in the live edit view on the main frame

**Screenshots**
![image](https://github.com/ted-marozzi/mermaid-flow-feedback/assets/126403501/1d82888d-bb11-42c7-8df1-fa3c3ef685e0)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser edge

**Additional context**

**Comment by ted-marozzi** [comment]
Correct, currently only flowchart diagrams are supported. Reopening to track support!
